pandas/_libs/lib.pyx

```diff
@@ -1250,6 +1252,23 @@ def is_pyarrow_array(obj):
     return False
 
 
+def is_null_slice(obj):
```

Comment:
does this need changing following discussion in the other PR?

Reply:
Yep - this can be removed. Will get to it on next pass